examples/src/test/java/cucumber/runtime/arquillian/CukesInBellyWithByClassConfigTest.java

```diff
@@ -1,51 +1,50 @@
-package cucumber.runtime.arquillian;

```

Comment:
I'm pretty sure I removed some imports from this file, not sure why the diff is not showing them